package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "elmi-to-json",
```

Comment:
❓ will the name be `elmi-to-json`, or `elm-interface-to-json`?

Reply:
I would like to use `elmi-to-json`